packages/flutter/lib/src/widgets/actions.dart

```diff
@@ -92,6 +94,19 @@ abstract class Action<T extends Intent> with Diagnosticable {
   /// [notifyActionListeners] to notify any listeners of the change.
   bool isEnabled(covariant T intent) => true;
 
+  /// Indicates whether this action should treat key events mapped to this
+  /// action as being "handled" when it is invoked via the key event.
+  ///
+  /// If the key is handled, then no other key event handlers in the focus chain
+  /// will receive the event.
+  ///
+  /// If the key event is not handled, it will be passed back to the engine, and
+  /// continue to be processed there, allowing text fields and non-Flutter
+  /// widgets to receive the key event.
+  ///
+  /// The default implementation returns true.
+  bool shouldHandleKey(covariant T intent) => true;
```

Comment:
nit: should we rename this to `consumesKey` as in "If this action is invoked, it will fully consume the key and give nobody else a change to react to it"? If it doesn't consume the key, then other platform handlers can have it.

Reply:
OK, sounds good.  Done.